### DISCUSSION TITLE
secret newgrounds medals not working (?)

### DISCUSSION THREAD

**Discussion by bokuwadoctor** [FunkinCrew/Funkin#1486] [Q&A] [answered]
I know how both of these medals work but I have no clue if the code that makes these unlockable is in the game yet.

![chrome_38cKL6yWJz](https://user-images.githubusercontent.com/77166605/107681141-b1c9eb00-6c6c-11eb-8d74-114f1ace4ba6.png)

**Comment by suprstarrd** [comment]
Will put it here at this point: I can confirm that these are not unlockable anywhere in the game. 

I was the *only* one to unlock them because Newgrounds API magic which I won't delve into-- but then it got removed, I assume because of what I did

anyways @ninjamuffin99 your API keys are publicly visible in the compiled JavaScript please obfuscate/encrypt or something 🙏

**Comment by ninjamuffin99** [comment, marked as answer]
was gon be for when i was in the mood to add more medals, but then i changed my mind

api keys will be regenerated and sorted out when i overhaul scoreboards and scoring system and all that, for now i dont care too much lol

**Reply by AbnormalPoof** [reply]
i have an idea for a medal
maybe one where you beat milf without losing your combo

**Reply by bokuwadoctor** [reply]
fucking demented

**Reply by AbnormalPoof** [reply]
lmao

**Reply by AbnormalPoof** [reply]
that was a joke btw-

**Reply by GuglioREAL** [reply]
i mean milf isnt that hard

**Reply by isakube** [reply]
> i have an idea for a medal
> maybe one where you beat milf without losing your combo

Funkin' Legend

Difficulty: 3-4
Beat M.I.L.F without missing
